CONTRIBUTING.md

```diff
@@ -1,3 +1,50 @@
+## Set Up a Development Environment
+
+1. `$ git clone git@github.com:ruby/debug.git`
+2. `$ bundle install`
+3. `$ rake` - this will
+    - Compile the C extension locally (which can also be done solely with `rake compile`).
+    - Run tests.
+    - Re-generate `README.md`.
+
+If you spot any problem, please open an issue.
+
+## Run Tests
```

Comment:
@ono-max to you want to add any tips on running tests?

Reply:
You can change this suggestion, but I wanna add this part.
```suggestion
## Run Tests

### Run specific test
$ ruby bin/test-unit.rb debug/break_test.rb
```